pkg/virt-controller/watch/vm_test.go

```diff
@@ -1161,6 +1162,19 @@ var _ = Describe("VirtualMachine", func() {
 					})
 				}
 
+				k8sClient.PrependReactor("get", "persistentvolumeclaims", func(action testing.Action) (handled bool, obj runtime.Object, err error) {
+					get, ok := action.(testing.GetAction)
+					Expect(ok).To(BeTrue())
+					if dv.Spec.Source != nil {
+						Expect(get.GetNamespace()).To(Equal(dv.Spec.Source.PVC.Namespace))
+						Expect(get.GetName()).To(Equal(dv.Spec.Source.PVC.Name))
+					} else {
+						Expect(get.GetNamespace()).To(Equal(ds.Spec.Source.PVC.Namespace))
+						Expect(get.GetName()).To(Equal(ds.Spec.Source.PVC.Name))
+					}
```

Comment:
Maybe get name & ns in the if/else and follow with common Expects. It took me a few seconds to see the dv vs. ds.